util/util_windows.go

```diff
@@ -26,3 +29,47 @@ func RunCommand(command string) (string, string, error) {
 
 	return string(outBuffer.Bytes()), string(errBuffer.Bytes()), nil
 }
+
+// FilesystemInfo XXX
+type FilesystemInfo struct {
+	PercentUsed string
+	KbUsed      float64
+	KbSize      float64
+	KbAvailable float64
+	Mount        string
+	Label        string
+	VolumeName  string
+	FsType      string
+}
```

Comment:
[IMHO] `type FilesystemInfo` is only used by `util/windows/filesystem`. I think it's better to move to `util/windows/filesystem.go`.